Sources/NIO/SelectorEpoll.swift

```diff
@@ -93,6 +93,35 @@ extension SelectorEventSet {
     }
 }
 
+@usableFromInline struct EPollUserData {
```

Comment:
FWIW without the context of the rest of the PR I don't think it's immediately obvious why we're packing/unpacking from a `UInt64`, a brief comment might be useful.

Reply:
done